src/components/db.ts

```diff
@@ -1,5 +1,12 @@
 import sqlite3 from 'sqlite3';
 import { open, Database } from 'sqlite';
+
+import { initSuggestionsTable } from './suggestions';
```

Comment:
ya i may have done smth here to fix a conflict, but plz pull latest bc the format of db.ts changed to avoid cyclic imports